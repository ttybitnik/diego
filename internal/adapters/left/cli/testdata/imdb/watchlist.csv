Position,Const,Created,Modified,Description,Title,URL,Title Type,IMDb Rating,Runtime (mins),Year,Genres,Num Votes,Release Date,Directors,Your Rating,Date Rated
222,tt5537002,2023-12-25,2023-12-25,,Killers of the Flower Moon,https://www.imdb.com/title/tt5537002/,movie,7.8,206,2023,"Crime, Drama, History, Mystery, Romance, Western",139774,2023-05-20,Martin Scorsese,,
223,tt21027780,2023-12-25,2023-12-25,,Kuolleet lehdet,https://www.imdb.com/title/tt21027780/,movie,7.6,81,2023,"Comedy, Drama",6786,2023-05-23,Aki Kaurismäki,,
224,tt26255088,2023-12-25,2023-12-25,,"Elis & Tom, só tinha de ser com você",https://www.imdb.com/title/tt26255088/,movie,7.9,100,2023,Documentary,240,2023-08-24,"Roberto de Oliveira, Jom Tob Azulay",,