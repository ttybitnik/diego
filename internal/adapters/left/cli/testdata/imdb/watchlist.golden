- name: Killers of the Flower Moon
  directors: Martin Scorsese
  year: "2023"
  created: "2023-12-25"
  position: "222"
  url: https://www.imdb.com/title/tt5537002/
- name: Kuolleet lehdet
  directors: Aki Kaurismäki
  year: "2023"
  created: "2023-12-25"
  position: "223"
  url: https://www.imdb.com/title/tt21027780/
- name: Elis & Tom, só tinha de ser com você
  directors: Roberto de Oliveira, Jom Tob Azulay
  year: "2023"
  created: "2023-12-25"
  position: "224"
  url: https://www.imdb.com/title/tt26255088/
